pkg/openevec/edenConfig.go

```diff
@@ -157,6 +157,14 @@ func ConfigList() error {
 	return nil
 }
 
+func ConfigFromViperValidate() error {
```

Comment:
Why not name it ValidateConfigFromViper?

Reply:
Make sense, will rename